src/components/PopupWithForm.js

```diff
@@ -8,6 +11,16 @@ export class PopupWithForm extends Popup {
   close() {
     super.close();
     this.form.reset();
+    //
+
+    const errorMessages = this.form.querySelectorAll(".form__input-error");
+    errorMessages.forEach((errorMessage) => {
+      errorMessage.classList.remove("form__input-error_active");
+    });
+    const inputs = this.form.querySelectorAll(".form__input");
+    inputs.forEach((input) => {
+      input.classList.remove("form__input_type_error");
+    });
```

Comment:
```js

    const errorMessages = this.form.querySelectorAll(".form__input-error");
    errorMessages.forEach((errorMessage) => {
      errorMessage.classList.remove("form__input-error_active");
    });
    const inputs = this.form.querySelectorAll(".form__input");
    inputs.forEach((input) => {
      input.classList.remove("form__input_type_error");
    });
```

Please, delete it

Now only class `FormValidator` should **`disable/enable`** submit buttons and remove errors from the inputs (if you do it)